build_tools/llvm_version.txt

```diff
@@ -1 +1 @@
-00128a20eec27246719d73ba427bf821883b00b4
+3f9cabae0029bcbe88835aaa4c417ce41e584fb1
```

Comment:
Revert? I'd guess we're ahead of this pin by now.

Reply:
Done.